KtuPharma/PharmaAPI
# Feature/test migration

### Things achieved with this MR:
* Establish connection with DB using DbContext
* Create medicament model to test it out
* Create migrations and seeds to populate DB with medicaments data
* Create test endpoint and associated DTOs to ensure EF is working properly.

## How to test:
1. Make sure all dependencies are installed using `dotnet restore` command.
2. Launch API project
3. Call `GET` method on `http://localhost:5000/api/v1/medicaments` using Postman. Make sure correct headers are set.
